rust_src/Cargo.toml

```diff
@@ -14,6 +14,7 @@ remacs-lib = { version = "0.1.0", path = "remacs-lib/" }
 remacs-macros = { version = "0.1.0", path = "remacs-macros" }
 libc = "0.2"
 rand = "0.3.15"
+md5 = "0.3.5"
```

Comment:
We should consider using the `md-5` and `sha-1` crates which have the same interface (and maintainer) as sha2 and a host of other digest crates. (Yes, the naming is unfortunate.)